.travis.yml

```diff
@@ -1,10 +1,18 @@
 language: rust
-rust: nightly
+rust:
+ # Jemalloc prefixing change between these two versions
+ - nightly-2016-02-14
+ - nightly-2016-02-17
+
+ - nightly
+ - beta
+ - stable
+
 notifications:
   webhooks: http://build.servo.org:54856/travis
 
 script:
  - cargo test
- - "[ $TRAVIS_RUST_VERSION = nightly ] || cargo test --features unstable"
- - "[ $TRAVIS_RUST_VERSION = nightly ] || cargo test --manifest-path plugin/Cargo.toml"
+ - "[ $TRAVIS_RUST_VERSION != nightly ] || cargo test --features unstable"
+ - "[ $TRAVIS_RUST_VERSION != nightly ] || cargo test --manifest-path plugin/Cargo.toml"
```

Comment:
Right, but the point of having these two dated version is to test the build script’s version parsing and the linking to `je_malloc_usable_size`. These versions are not special as far as the `unstable` feature and the plugin are concerned.